packs/feats/final-form.json

```diff
@@ -0,0 +1,49 @@
+{
+    "_id": "NIwocFnVpyzjeNUC",
+    "img": "icons/sundries/books/book-red-exclamation.webp",
+    "name": "Final Form",
+    "system": {
+        "actionType": {
+            "value": "action"
+        },
+        "actions": {
+            "value": 3
+        },
+        "category": "ancestry",
+        "description": {
+            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You embrace your monstrous blood and make it your own, screaming out to the heavens and hells alike in rage, pain, and defiance. Reality buckles and twists under your sudden power—each tiefling has a unique transformation, but it might include sulfurous flames, choruses of shrieks, or twisting, serpentine lightning. Every creature within 10 feet of you is @UUID[Compendium.pf2e.conditionitems.Item.Deafened] and @UUID[Compendium.pf2e.conditionitems.Item.Dazzled] for 1 round unless they succeed at a @Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)]{Fortitude save} using your class DC or spell DC, whichever is higher. Your final form has the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Divine Vessel]</em> for a deity that shares an alignment with your fiendish lineage. Your final form resembles a creature from your fiendish lineage, and you remain in that form for 1 minute. While in your final form, you can cast <em>@UUID[Compendium.pf2e.spells-srd.Item.Chilling Darkness]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Fireball]</em>, and <em>@UUID[Compendium.pf2e.spells-srd.Item.Lightning Bolt]</em> as 6th-rank divine innate spells, once each per use of this feat.</p>\n<p>When you emerge from your final form, you become @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 3}. In addition, if you have this feat, you gain weakness 10 to good even when not in your final form, and you take good damage even if you aren't evil.</p>\n<hr />\n<p><strong>Special</strong> At the GM's discretion, the specific elements of this feat can be adjusted to better fit the tiefling's lineage—for example, a GM might decide that a particular beastbrood tiefling could cast a 6th-rank <em>@UUID[Compendium.pf2e.spells-srd.Item.Phantasmal Killer]</em> instead of <em>fireball</em>.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You embrace your monstrous blood and make it your own, screaming out to the heavens and hells alike in rage, pain, and defiance. Reality buckles and twists under your sudden power—each tiefling has a unique transformation, but it might include sulfurous flames, choruses of shrieks, or twisting, serpentine lightning. Every creature within 10 feet of you is @UUID[Compendium.pf2e.conditionitems.Item.Deafened] and @UUID[Compendium.pf2e.conditionitems.Item.Dazzled] for 1 round unless they succeed at a @Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)] save using your class DC or spell DC, whichever is higher. Your final form has the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Divine Vessel]</em> for a deity that shares an alignment with your fiendish lineage. Your final form resembles a creature from your fiendish lineage, and you remain in that form for 1 minute. While in your final form, you can cast <em>@UUID[Compendium.pf2e.spells-srd.Item.Chilling Darkness]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Fireball]</em>, and <em>@UUID[Compendium.pf2e.spells-srd.Item.Lightning Bolt]</em> as 6th-rank divine innate spells, once each per use of this feat.</p>\n<p>When you emerge from your final form, you become @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 3}. In addition, if you have this feat, you gain weakness 10 to good even when not in your final form, and you take good damage even if you aren't evil.</p>\n<hr />\n<p><strong>Special</strong> At the GM's discretion, the specific elements of this feat can be adjusted to better fit the tiefling's lineage—for example, a GM might decide that a particular beastbrood tiefling could cast a 6th-rank <em>@UUID[Compendium.pf2e.spells-srd.Item.Phantasmal Killer]</em> instead of <em>fireball</em>.</p>"
```